job_spec/INSAR_GAMMA_HIGH_MEMORY.yml

```diff
@@ -0,0 +1,102 @@
+INSAR_GAMMA:
```

Comment:
Maybe? I don't actually know if this should change.
```suggestion
INSAR_GAMMA_HIGH_MEMORY:
```

Reply:
The job_type you give to the API is driven by the top-level dictionary key, not the file name. This lets azdwr-hyp3 continue to use the same job name (`INSAR_GAMMA`) but get the higher memory reservation.

We do the same thing with AUTORIFT.yml and AUTORIFT_ITS_LIVE.yml, so jobs in both edc-prod and hyp3-autorift are named `AUTORIFT`, but are run with different configurations.

Reply:
Cool, thanks for clearing that up.